python/demos/UserContributions/templates/user_contributions.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" content="width=device-width, initial-scale=1.0" name="viewport">
+
+    <!-- Page title -->
+    <title>User Contributions - Demo app</title>
+
+    <!-- Bootstrap CSS -->
+    <link href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css"
```

Comment:
For the sake of demo app, might not be a good idea to link to an external css file that lives on Toolforge..